train_wrapper.py

```diff
@@ -1,34 +1,157 @@
 """Functionality for locally training files."""
+import argparse
+import os
+from typing import List
+
 import torch
 import torch.nn as nn
 from torch.utils.data import DataLoader
 
 from app.ml_models.rnn.data_tools import WordLevelDataset
 from app.ml_models.rnn.vocabulary import Vocabulary
 from app.ml_models.rnn.train import train
+from app.ml_models.rnn.loaded_rnn_model import init_torch_model_from_path
 from app.ml_models.rnn.rnn_model import RNNAnna
 
 
-def run():
+def _run(**kwargs):
```

Comment:
I like the addition of the leading underscore. Should we also become more strict with the return types on all of our functions?
```suggestion
def _run(**kwargs) -> None:
```
Although we're gonna be adding a bunch of "-> None"s